middlewares/modelValidator.js

```diff
@@ -0,0 +1,70 @@
+import models from "../models";
```

Comment:
@niomwungeri-fabrice, Thanks for the brilliant ideas to implement the validator!
But ModelValidator doesn't follow the single responsibility principle...
I would suggest you to create a separate validator for each model.
And we have already had article validator middleware, there is no need to recreate it.

Reply:
@espoirMur, Thanks, for the suggestion, unless you are suggesting to rename the file to reflect the functions it contains otherwise the class is doing validation and all those methods inside it are doing validation.
Thanks